tests/core/bures_continuous_barycenter_test.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Apple
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for Continuous barycenter of GMMs with Bures cost."""
+
+import jax.numpy as jnp
+from absl.testing import absltest, parameterized
+
+from ott.core import bar_problems, continuous_barycenter
+from ott.geometry import costs
+
+
+class Barycenter(parameterized.TestCase):
+
+  def setUp(self):
+    super().setUp()
+
+  @parameterized.product(
```

Comment:
maybe a simpler formulation could be to append this def test_bures_barycenter... directly at the bottom of continuous_barycenter_test.py